app/models/repo.js

```diff
@@ -130,18 +130,47 @@ Repo.reopen({
     return array;
   }.property(),
 
+  crons: function() {
+    var array, builds, id;
+    id = this.get('id');
+    builds = this.store.filter('build', {
+      event_type: 'cron',
+      repository_id: id
+    }, function(b) {
+      return b.get('repo.id') + '' === id + '' && b.get('eventType') === 'cron';
+    });
+    array = ExpandableRecordArray.create({
+      type: 'build',
+      content: Ember.A([])
+    });
+    array.load(builds);
+    id = this.get('id');
+    array.observe(builds);
+    return array;
+  }.property(),
```

Comment:
My understanding is that this code sets up a pseudo-relationship between a `repo` and its `crons`. Normally one would do this with `hasMany` but that appears to not be possible. So this technique observes all known `build`s within the Ember Data store and selects the ones that belong to this `repo`.

Given that `filter` no longer exists on the Ember Data store, it looks like [the Ember Way is to use computed properties with `peekAll`](https://github.com/ember-data/ember-data-filter#i-dont-want-to-enable-this-feature-due-to-its-heavy-churn). I imagine that would look like this:

```
builds: Ember.computed(function() {
  return this.store.peekAll('build');
}),

crons: Ember.computed('id', 'builds.@each.repo', function() {
  return this.get('builds').filterBy('eventType', 'cron').filterBy('repo.id', this.get('id'));
})
```

I’m not quite sure about that, but see how it’s more concise? It seems easier to understand to me, and it’s more idiomatic.

I’m not sure what the role of the `event_type` and `repository_id` filters are versus the `function(b)` filter, they appear to be identical? To me, this is the kind of code that pretty much demands an automated test.